test/e2e/pod_top_test.go

```diff
@@ -127,6 +129,13 @@ var _ = Describe("Podman top", func() {
 		session.WaitWithDefaultTimeout()
 		Expect(session.ExitCode()).To(Equal(0))
 
+		for i := 0; i < 10; i++ {
+			fmt.Println("Waiting for containers to be running .... ")
+			if podmanTest.NumberOfContainersRunning() == 2 {
+				break
+			}
+			time.Sleep(1 * time.Second)
+		}
```

Comment:
If tests are running in parallel, this loop seems ripe for causing headaches.  Would it make sense to setup and wait for some test-specified or random-named containers instead?  Unless there's zero chance this will ever be able to count another test's containers.

Also, when the wait is successful, it might be handy to toss a debugging line in that reports how long it actually waited.  That historical data can be really handy if/when a time comes where the timeout length needs adjusting.

Reply:
we are no worse off than before as we dont do error catching in our tests, least not here.  so im ok with this as a stop gap.